sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -142,6 +142,25 @@ on which actual message transmission takes place. The namespace often serves as
 * A **[subscription][subscription_concept]** receives messages from a topic. Each subscription is independent and
   receives a copy of the message sent to the topic.
 
+To interact with these resources, one should be familiar with the following SDK concepts:
+
+* All the Service Bus Clients are created using `ServiceBusClientBuilder`.
```

Comment:
These are extraneous and implied from the "Authenticate the client" and samples. These are usually Key Concepts about using the service itself. (ie. What is a queue? What is a topic?)